libraries/src/HTML/HTMLHelper.php

```diff
@@ -1219,6 +1219,58 @@ public static function calendar($value, $name, $id, $format = '%Y-%m-%d', $attri
 		return LayoutHelper::render('joomla.form.field.calendar', $data, null, null);
 	}
 
+	/**
+	 * Add data-attributes for the inline editing.
+	 *
+	 * @param   string  $component  com_content, com_fields, etc
+	 * @param   string  $controller  Article, field, etc
+	 * @param   mixed   $attribs  Additional HTML attributes required to make Ajax Request
+	 *                            Example: In case of Article title,
+	 *                                      $attribs = [a_id => 23].
+	 *
+	 * @return  string  Formatted Data attributes
+	 *
+	 * @since __DEPLOY_VERSION__
+	 */
+	public static function convertToDataAttributes(string $component, string $controller, $attribs = [])
+	{
+		// Load script
+		$app = Factory::getApplication();
+		$document = Factory::getDocument();
+		$wa = $document->getWebAssetManager();
+		$wa->useStyle('webcomponent.inline-editing');
+
+		if ($app->get('editor', '') == 'tinymce')
```

Comment:
I have added two scripts.
1. 'inline-editing-tinymce.js' Uses tiny editor inline edition
2. 'inline-editing.js' Uses a text area.

It checks if the 'TinyMCE is the default editor. If true, it loads the tiny editor and the script to enable inline editing. Otherwise, It loads another script that adds a text area.

Question: Is this the right way to load the Tiny editor? Or is there another better way to load the tiny editor?
@roland-d @anibalsanchez @bembelimen

Reply:
Tiny editor plugin loads this file directly. So, I think that it is the right way.